cumulus/test/client/src/block_builder.rs

```diff
@@ -32,12 +37,19 @@ pub trait InitBlockBuilder {
 	/// valid for the test runtime.
 	///
 	/// You can use the relay chain state sproof builder to arrange required relay chain state or
-	/// just use a default one.
+	/// just use a default one. The relay chain slot in the storage proof
+	/// will be adjusted to align with the parachain slot to pass validation.
+	///
+	/// Returns the block builder and validation data for further usage.
 	fn init_block_builder(
 		&self,
 		validation_data: Option<PersistedValidationData<PHash, PBlockNumber>>,
 		relay_sproof_builder: RelayStateSproofBuilder,
-	) -> sc_block_builder::BlockBuilder<Block, Client>;
+	) -> (
+		sc_block_builder::BlockBuilder<Block, Client>,
+		PersistedValidationData<PHash, PBlockNumber>,
+		Slot,
+	);
```

Comment:
struct would be nice here. the tuple seems to be duplicated.

Reply:
Done!